recipes-kernel/linux/linux-mainline/arm/defconfig

```diff
@@ -1207,68 +1269,97 @@ CONFIG_NET_VENDOR_SOLARFLARE=y
 CONFIG_NET_VENDOR_SOCIONEXT=y
 CONFIG_NET_VENDOR_STMICRO=y
 CONFIG_STMMAC_ETH=y
+# CONFIG_STMMAC_SELFTESTS is not set
 CONFIG_STMMAC_PLATFORM=y
 # CONFIG_DWMAC_DWC_QOS_ETH is not set
 CONFIG_DWMAC_GENERIC=y
 CONFIG_DWMAC_SUNXI=y
 CONFIG_DWMAC_SUN8I=y
+# CONFIG_DWMAC_INTEL_PLAT is not set
 CONFIG_NET_VENDOR_SYNOPSYS=y
 # CONFIG_DWC_XLGMAC is not set
 # CONFIG_NET_VENDOR_VIA is not set
 # CONFIG_NET_VENDOR_WIZNET is not set
-CONFIG_MDIO_DEVICE=y
```

Comment:
Same as above ..
Does network phy work without these options ?

Reply:
Yes, it does!

Reply:
ok